ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-bgp-vrfs.j2

```diff
@@ -67,7 +67,11 @@
 {%                     endif %}
       neighbors:
 {%                     if configure_mlag_ibgp_peering %}
-        {{ switch.mlag_peer_l3_ip | arista.avd.default(switch.mlag_peer_ip) }}:
+{%                         if vrf.mlag_ibgp_peering_ipv4_pool is arista.avd.defined %}
+{%                             set mlag_ibgp_peering_ip = namespace() %}
+{%                             include switch.ip_addressing.mlag_ibgp_peering_ip_secondary %}
+{%                         endif %}
+        {{ mlag_ibgp_peering_ip.peer_ip | arista.avd.default(switch.mlag_peer_l3_ip, switch.mlag_peer_ip) }}:
```

Comment:
`mlag_ibgp_peering_ip.peer_ip ` will always have the same value whether it is primary or secondary switch we are working on (cf generated configs)